src/components/Layout/Header/Header.tsx

```diff
@@ -55,7 +64,23 @@ export const Header = () => {
         zIndex='banner'
         top={0}
       >
-        <HStack width='full' px={4}>
+        {walletInfo?.deviceId === 'DemoWallet' && (
+          <Box
+            bg='blue.500'
+            width='full'
+            height='2.5rem'
```

Comment:
It feel like that could be nitpick but it's not:

you should never set a fixed height on any elements, you tested it using english or another language, and it works pretty fine on mobile, but some languages could result in bigger text width and it would completely be broken

Reply:
Prefer a minHeight, even if it's not perfect with other calculation or whatever, it will be better than nothing

Reply:
This is fair enough. I also did some refactor on the text elements (within this Box element) so they wrap more nicely on super small screens.